app/code/Magento/Customer/Model/ResourceModel/Customer.php

```diff
@@ -21,9 +22,9 @@
 class Customer extends \Magento\Eav\Model\Entity\VersionControl\AbstractEntity
 {
     /**
-     * @var \Magento\Framework\Validator\Factory
+     * @var \Magento\Framework\Stdlib\DateTime
      */
-    protected $_validatorFactory;
+    protected $dateTime;
```

Comment:
Why did you moved it above?

Reply:
Sort in alphabetical order ;) I can revert this change if you request it. Thanks

Reply:
It’s better to keep only needed changes in order to prevent conflicts, so please revert it.

Reply:
yes. Now I understand this. Thank you